src/frontends/tensorflow_common/src/op/atan2.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+
+#include "common_op_table.hpp"
+#include "openvino/op/atan.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/convert_like.hpp"
+#include "openvino/op/divide.hpp"
+#include "openvino/op/equal.hpp"
+#include "openvino/op/greater.hpp"
+#include "openvino/op/greater_eq.hpp"
+#include "openvino/op/less.hpp"
+#include "openvino/op/logical_and.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/select.hpp"
+#include "openvino/op/subtract.hpp"
+
+
+using namespace std;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+OutputVector translate_Atan2_op(const NodeContext& node) {
+    default_op_checks(node, 2, {"Atan2"});
+    auto y = node.get_input(0);
+    auto x = node.get_input(1);
+
+    // handle the first condition : x>0
+    auto div_y_x = make_shared<Divide>(y, x);
```

Comment:
you need to add version of operation `v1`: `make_shared<v1::Divide>(y, x)`.
For other operations below please add as well. See documentation to get the version: https://docs.openvino.ai/2023.2/groupov_ops_cpp_api.html